### ISSUE TITLE
Bun crashed running prisma init

### ISSUE THREAD

**Issue by coredoesdev** [oven-sh/bun#15916]
### How can we reproduce the crash?

1. Run `bunx --bun prisma init --datasource-provider postgres` in any package

### Relevant log output

```shell
Bun Canary v1.1.41-canary.29 (b8f28ed8) Linux x64
Linux Kernel v6.12.4 | glibc v2.40
CPU: sse42 popcnt avx avx2
Args: "node" "/home/core/prj/e3t/console.e3t.cc/node_modules/.bin/prisma" "init" "--datasource-provider" "postgres"
Features: jsc tsconfig(2) tsconfig_paths 
Builtins: "bun:main" "node:fs" "node:string_decoder" "node:util/types" 
Elapsed: 289ms | User: 217ms | Sys: 54ms
RSS: 1.02GB | Peak: 0.16GB | Commit: 1.02GB | Faults: 0

panic(main thread): Segmentation fault at address 0x0
Crashed while visiting /home/core/prj/e3t/console.e3t.cc/node_modules/@prisma/get-platform/dist/chunk-6HZWON4S.js
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:

 https://bun.report/1.1.41/ln2b8f28edChgg8//DoqkumE+8oP06rjoDqt4miDqz20hD+s45gDy364gDkhn6gD40vs+D_A2AA
```

### Stack Trace (bun.report)

Bun v1.1.41 ([`b8f28ed`](<https://github.com/oven-sh/bun/tree/b8f28ed8afd1c2b60568b2b0158d39a674178027>)) on linux x86_64 [RunAsNodeCommand]

Segmentation fault at address 0x00000000

- [`ld-temp.o:0`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/ld-temp.o#L0>): `JSC::JSCallbackObject<JSC::JSNonFinalObject>::create`
- `??`
- [`js_parser.zig:10003`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/js_parser.zig#L10003>): `src.js_parser.NewParser_`
- [`resolver.zig:2953`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/resolver/resolver.zig#L2953>): `src.resolver.resolver.Resolver.matchTSConfigPaths`
- [`javascript.zig:0`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/src/bun.js/javascript.zig#L0>): `src.bun.js.javascript.VirtualMachine.printErrorlikeObject__anon_122122`
- [`io.zig:360`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027//opt/homebrew/var/buildkite-agent/builds/darwin-aarch64-studio-2-18/bun/bun/vendor/zig/lib/std/io.zig#L360>): `src.bun.js.javascript.VirtualMachine.processFetchLog`
- [`mem.zig:1241`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027//opt/homebrew/var/buildkite-agent/builds/darwin-aarch64-studio-2-18/bun/bun/vendor/zig/lib/std/mem.zig#L1241>): `src.watcher.NewWatcher__struct_19580.addFile__anon_117134`
- [`Allocator.zig:240`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027//opt/homebrew/var/buildkite-agent/builds/darwin-aarch64-studio-2-18/bun/bun/vendor/zig/lib/std/mem/Allocator.zig#L240>): `WebWorker__create`
- [`JSObjectInlines.h:188`](<https://github.com/oven-sh/bun/blob/b8f28ed8afd1c2b60568b2b0158d39a674178027/./build/release-ci/cache/webkit-3845bf370ff4e9a5c0b96036255142c7904be963/include/JavaScriptCore/JSObjectInlines.h#L188>): `Bun::populateESMExports`

**Comment by DonIsaac** [comment]
I cannot reproduce this on MacOS with version `1.1.40` or a build off `main`. I ran `bunx --bun prisma init --datasource-provider postgres` in a fresh repo, created with `bun init`.

It's worth noting that `postgres` is not a valid datasource provider: you need to use `postgresql`. On my end the former name errored and the latter one initalized prisma - all as expected.

Questions:
1. If you run `bunx --bun prisma init --datasource-provider postgresql`, does it crash?
2. Can you reproduce the crash on the latest stable release of bun? Looks like you're using a canary build

**Comment by Jarred-Sumner** [comment]
Fixed by @dylan-conway in #15911
